build-aux/snap/snapcraft.yaml

```diff
@@ -232,6 +232,8 @@ parts:
       - libudev-dev
       - libcap-dev
       - libseccomp-dev
+      - libapparmor-dev
```

Comment:
Hmm, should we not use apparmor headers that we get from the part? Maybe something is misconfigured?

Reply:
good point